language/vm/vm-runtime/src/libra_vm.rs

```diff
@@ -592,6 +591,18 @@ pub fn is_allowed_script(publishing_option: &VMPublishingOption, program: &[u8])
     }
 }
 
+/// Convert the transaction arguments into move values.
```

Comment:
@runtian-zhou I moved this here given it is the only place where it is used. It should be fine and I'll let you re-organize in the future as you see fit